tsconfig.json

```diff
@@ -4,11 +4,11 @@
     "allowJs": true,
     "allowNonTsExtensions": true,
     "experimentalDecorators": true,
+    "useDefineForClassFields": false,
```

Comment:
not sure why it has been moved up but ok

Reply:
by mistake, will revert